remoteworkitem/tracker_repository_test.go

```diff
@@ -55,6 +58,33 @@ func (test *TestTrackerRepository) TestTrackerCreate() {
 	assert.NotNil(t, tracker2)
 }
 
+func (test *TestTrackerRepository) TestExistsTracker() {
+	t := test.T()
+	resource.Require(t, resource.Database)
+
+	t.Run("tracker exists", func(t *testing.T) {
+		t.Parallel()
+		// given
+		tracker, err := test.repo.Create(context.Background(), "http://api.github.com", ProviderGithub)
+		assert.Nil(t, err)
```

Comment:
> Why is it better ?

because it makes no sense to assert on some return value if you know the result is garbage by inspecting the error.